src/csharp/Pester/OutputConfiguration.cs

```diff
@@ -59,6 +62,22 @@ public StringOption Verbosity
             }
         }
 
+        public StringOption StackTraceVerbosity
```

Comment:
Is this a good name for the option? I'm happy to call it something more suitable if needed.

Reply:
> I may also be doing the **Filtered** option incorrectly. Will need to review the behavior from https://github.com/pester/Pester/blob/main/src/functions/Output.ps1#L423 to see If I need to change this, since currently I am just outputting the lines from the trace that are picked up as Pester internal functions.
>
> I guess I might need some help defining pester specific lines that need to be filtered, since my current Filtered option seems to be leaving a lot out, and some lines from the **Full** output look like they should belong in **Filtered**.

It not only removes Pester lines, but in case of Should, it removes everything before the call to Should. That is maybe why it seems to leave out a lot more than you'd expect. 
https://github.com/pester/Pester/blob/main/src/functions/Output.ps1#L441

IMHO you are doing it right.

Reply:
Ah thought that would be the case, thanks for clarifying 🙂 .